### PR TITLE
Typing in an input should not trigger the CTRL LEFT / RIGHT shortcuts

### LINKED ISSUE #337: Typing in an input should not trigger the CTRL LEFT / RIGHT shortcuts
As a User
I would like Studio to not trigger the CTRL LEFT / RIGHT shortcuts when I'm typing in a Text Input
So that CTRL + SHIFT + LEFT / RIGHT wont trigger them and break my flow

## Steps to reproduce
- GIVEN that I've loaded a project,
- THEN gone to the Texts management UI,
- WHEN I'm editing a text entry in the list,
- AND I'm using the CTRL LEFT / RIGHT shortcuts
- THEN Studio will navigate to the previous or next CSV instead

## Expected behaviour
- GIVEN that I've loaded a project,
- THEN gone to the Texts management UI,
- WHEN I'm editing a text entry in the list,
- AND I'm using the CTRL LEFT / RIGHT shortcuts
- THEN my cursor should go to the previous / next word
See the reaction by using Notepad, this is a basic typing shortcut.

### PR DESCRIPTION
**Pull request by Zodiarche**
## Description

If you're in an input, whether it's a search or an input to modify text, you won't be able to use shortcuts.

## Tests to perform

- Put yourself in the text change input, make a shortcut, it won't happen
- Put yourself in the research input, make a shortcut, it won't happen
- Be on the page, make a shortcut, it will happen

### CODE REVIEW

File: src/views/components/textmanagement/TextControlBar.tsx [hunk 1]
```diff
@@ -31,7 +31,14 @@ export const TextControlBar = ({ dialogsRef }: TextControlBarProps) => {
   // Definition of the control bar shortcuts
   const shortcutMap = useMemo<StudioShortcutActions>(() => {
     // No shortcut if an editor is opened
-    const isShortcutEnabled = () => dialogsRef?.current?.currentDialog === undefined;
+    const isShortcutEnabled = () => {
+      const activeElement = document.activeElement;
+      if (activeElement && activeElement.tagName === 'INPUT') return false;
```
Comment (Palbolsky) [inline src/views/components/textmanagement/TextControlBar.tsx]: Il faut également ajouter `TEXTAREA`  car dans la page de traduction on utilise des MultiLineInput et non des Input (les MultiLineInput sont des textareas)

Comment (Zodiarche) [reply src/views/components/textmanagement/TextControlBar.tsx]: Fixed

**Review by Palbolsky — APPROVED** [review]

File: src/views/components/textmanagement/TextControlBar.tsx (hunk 1 above)
Comment (Palbolsky) [reply src/views/components/textmanagement/TextControlBar.tsx]: Parfait :)

**Comment by NuriYuri** [comment]
Additional test: When pressing CTRL+LEFT in the input, the caret should go to the beginning of the Input line. When pressing CTRL+Right it should go at the end of the input line. (You can see this behaviour in any inputs on your browser, including the very same input I'm using to type this message).

Note: Windows behaviour might be different but it should have one ;)

**Comment by AerunDev** [comment]
> Additional test: When pressing CTRL+LEFT in the input, the caret should go to the beginning of the Input line. When pressing CTRL+Right it should go at the end of the input line. (You can see this behaviour in any inputs on your browser, including the very same input I'm using to type this message).
> 
> Note: Windows behaviour might be different but it should have one ;)

On Windows, the caret jump to the next or previous word.

**Review by AerunDev — APPROVED** [review]

**Pull request merged**